### PR TITLE
Create service plug ScenarioSourceListener

### PR DESCRIPTION
**Pull request by tatarynovdima**
i'm not sure about the line InputStream inputStream = getClass().getResourceAsStream("example.json");
where do I read Senarion from a file, maybe I should read them from a properties file?

### CODE REVIEW

File: src/main/java/executor/service/service/ScenarioSourceListenerImpl.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            JSONArray scenarios = new JSONArray(jsonStrBuilder.toString());
+            for (int i = 0; i < scenarios.length(); i++) {
+                JSONObject scenario = scenarios.getJSONObject(i);
+                String name = scenario.getString("name");
+                String site = scenario.getString("site");
+                JSONArray steps = scenario.getJSONArray("steps");
+
+                System.out.println("Scenario name: " + name);
+                System.out.println("Site: " + site);
+                System.out.println("Steps:");
```
Comment (Serhx4) [inline src/main/java/executor/service/service/ScenarioSourceListenerImpl.java:31]: I'm not sure we have to `println` anything here

**Review by AlexandrTokarenko — APPROVED** [review]

File: src/main/java/executor/service/service/ScenarioSourceListenerImpl.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import java.io.InputStream;
+import java.util.List;
+
+public class ScenarioSourceListenerImpl implements ScenarioSourceListener {
+
+    @Override
+    public void execute() {
+        try {
+            ObjectMapper objectMapper = new ObjectMapper();
+            InputStream inputStream = getClass().getResourceAsStream("example.json");
```
Comment (Serhx4) [inline src/main/java/executor/service/service/ScenarioSourceListenerImpl.java:16]: This one `InputStream` should be closed.
try-with-resources will fit well here

**Review by Duskmage2009 — APPROVED** [review]

**Review by Duskmage2009 — APPROVED** [review]

**Comment by Serhx4** [comment]
> i'm not sure about the line InputStream inputStream = getClass().getResourceAsStream("example.json"); where do I read Senarion from a file, maybe I should read them from a properties file?

Hey, @tatarynovdima, I missed this one question. We should read Scenarios from scenario.json file, which should be created into resources folder. An example of file is into your task's card

**Review by VatBane — APPROVED** [review]

**Review by artemkedyk — APPROVED** [review]

**Review by Serhx4 — APPROVED** [review]
Looks good!

**Review by bielik-b — APPROVED** [review]

**Review by antonRMNV — APPROVED** [review]

**Review by artemkedyk — APPROVED** [review]

**Pull request merged**